etc/picongpu/bash/mpiexec.tpl

```diff
@@ -24,8 +24,8 @@
 .TBG_author=${MY_NAME:+--author \"${MY_NAME}\"}
 .TBG_profile=${PIC_PROFILE:-"~/picongpu.profile"}
 
-# 4 gpus per node if we need more than 4 gpus else same count as TBG_tasks
-.TBG_gpusPerNode=$(if [ $TBG_tasks -gt 4 ] ; then echo 4; else echo $TBG_tasks; fi)
+# 8 gpus per node if we need more than 8 gpus else same count as TBG_tasks
```

Comment:
Our old number `4'and our new number `8` is a magic number which is randomly chosen. We should think about a more generic way (I have currently no idea).

Reply:
maybe we could just do
`ls -l /dev/nvidia* | wc -l` or so